e2e/test/scaling/provider.go

```diff
@@ -94,30 +116,25 @@ func (p *Provider) NumMasters() (int, error) {
 }
 
 func (p *Provider) NumWorkers() (int, error) {
-	customObject, err := p.hostFramework.G8sClient().ProviderV1alpha1().AzureConfigs("default").Get(p.clusterID, metav1.GetOptions{})
+	ctx := context.TODO()
```

Comment:
I would add the context to the function signature but then I'd have to change e2e tests repositories, make sure legacy aws tests works, [and...](https://i.kym-cdn.com/photos/images/newsfeed/000/284/529/e65.gif)